int8 GYRO_BIAS_ESTIMATION=1
int8 HARD_IRON_AUTO_CALIBRATION=32
int8 SOFT_IRON_AUTO_CALIBRATION=64
int8 flags
---
bool success
